opm/common/utility/TimeService.hpp

```diff
@@ -45,6 +45,7 @@ namespace Opm {
 
     std::time_t mkdatetime(int in_year, int in_month, int in_day, int hour, int minute, int second);
     std::time_t mkdate(int in_year, int in_month, int in_day);
+    std::time_t portable_timegm(struct tm const* t);
```

Comment:
Can we write this declaration as
```C++
std::time_t portable_timegm(std::tm const* t);
```
(or maybe `... timegm(const std::tm* t)` for that matter) instead please?  We already include `<ctime>` here and have a declaration for `std::tm` in scope.